### PR TITLE
[IMP] l10n_co: reuse identification types from l10n_latam_invoice_doc…

### PR DESCRIPTION
**Pull request by jco-odoo**
…ument

We also set a country_id for e.g. foreign which we should not.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@vbe-odoo If you can check and test  @bfr-o for your info

**Comment by vbe-odoo** [comment]
Hi @jco-odoo now I can see more document types, and there's a difference between a foreign contact (like in Argentina) and with the Colombian Document Types.

This fix is working, but the issue was when trying to create an invoice with a foreign contact (export invoice), and here I can not test that. Let me know if then I can test it on the other PR (https://github.com/odoo/enterprise/pull/13220)

Thanks!
cc' @ren-odoo

**Comment by bfr-o** [comment]
@vbe-odoo You are correct, but we identified the mistake as a bug and so we had to fix it. Once this is merged, I will rebase my branch on the other PR and you will be able to test it.

File: addons/l10n_co/data/l10n_latam.identification.type.csv [hunk 1]
```diff
@@ -2,10 +2,11 @@ id,name,l10n_co_document_code,country_id:id,is_vat,sequence
 rut,NIT,rut,base.co,TRUE,1
 national_citizen_id,Cédula de ciudadanía,national_citizen_id,base.co,FALSE,2
 external_id,ID del Exterior,external_id,base.co,FALSE,3
```
Comment (bfr-o) [inline addons/l10n_co/data/l10n_latam.identification.type.csv:4]: Shouldn't we remove the country for `external_id` as well ?

**Comment by vbe-odoo** [comment]
> @vbe-odoo You are correct, but we identified the mistake as a bug and so we had to fix it for all involved versions. Once this is merged, I will rebase my branch on the other PR and you will be able to test it.

@bfr-o thanks for letting me know, I will be waiting for the merge so. bests.

**Comment by jco-odoo** [comment]
@csnauwaert As you are assigned round robin, can you r+ it?

**Comment by csnauwaert** [comment]
@robodoo r+

**Pull request closed without merging**